src/iexamine.cpp

```diff
@@ -2233,7 +2233,7 @@ void iexamine::harvest_tree_shrub(player &p, const tripoint &examp)
     }
     //if the fruit is not ripe yet
     if (calendar::turn.get_season() != g->m.get_ter_harvest_season(examp)) {
-        std::string fruit = item::nname(g->m.get_ter_harvestable(examp), 10);
+        std::string fruit = item::nname(g->m.get_ter_harvestable(examp).str(), 10);
```

Comment:
Yes, you're right. I overlooked it. Thank you.
The name is misleading and I'm going to change it.